test/test_jit.py

```diff
@@ -15398,7 +15398,7 @@ def add_autograd_test(
 
     # Disable complex tests
     # TODO: Add complex support for jit
-    if 'complex' in variant_name:
+    if 'complex' in variant_name or name in ['view_as_complex']:
```

Comment:
this test runs with jit because it takes as input a real tensor. The test fails because it returns a complex tensor and complex tensors are not supported on jit